src/gcp_scanner/crawler/storage_buckets_crawler.py

```diff
@@ -24,6 +24,8 @@
 class StorageBucketsCrawler(ICrawler):
   """Handle crawling of bucket names data."""
 
+  _config_depndency = True # Private Variable Denoting Config Files Needed
```

Comment:
I'd rephrase it to just "# Define that config file is needed"

Reply:
Updated the comment with the new commit.